src/docs/forms/components/RadioGroup/examples/05-ValueKeyandLabelKey.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import Form, {
+    RadioGroup
+} from 'doj-react-adminlte/Form';
+
+export default class InlineRadioGroup extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            colors: false,
```

Comment:
rename state to `color`. also, initial state values for non-boolean option values must be `null`